<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PHPQt5::PHPQt5QMakeBuildStepConfigWidget</class>
 <widget class="QWidget" name="PHPQt5::PHPQt5QMakeBuildStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="additionalArgumentsLabel">
       <property name="text">
        <string>Extra arguments:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="additionalArgumentsLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="commandLabel">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="commandTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
